app/views/gobierto_participation/processes/index.html.erb

```diff
@@ -4,20 +4,44 @@
   </strong>
 <% end %>
 
-<h1>Gobierto Participation processes index</h1>
+<%= content_for :body_attributes do %>
+  class="theme-participation gobierto_participation"
+<% end %>
 
-Processes:
+<div class="gradient-bg">
+  <div class="column">
+    <div class="pure-g">
+      <div class="pure-u-1 pure-u-md-1-3">
+        <h2 class="with_separator"><%= t('.ongoing_processes') %></h2>
+        <p class="soft"><%= t('.ongoing_processes_p') %></p>
+      </div>
 
-<ul>
-  <% @processes.each do |process| %>
-    <li><%= link_to process.title, gobierto_participation_process_path(process.slug) %></li>
-  <% end %>
-</ul>
+      <div class="pure-u-1 pure-u-md-2-3 groups">
+        <div class="pure-g three_by_three">
+          <% @processes.each do |process| %>
+            <%= render 'process_teaser', process: process, meta: false %>
```

Comment:
I think the `meta: true` parameter can be removed. Is something only used in the sandbox.